src/main/java/uk/ac/ebi/eva/evaseqcol/service/SeqColService.java

```diff
@@ -312,27 +313,33 @@ public SeqColComparisonResultEntity compareSeqCols(
         comparisonResult.putIntoArrays("b_only", seqColBUniqueAttributes);
         comparisonResult.putIntoArrays("a_and_b", seqColCommonAttributes);
 
-        // "elements" attribute | "total"
-        Integer seqColATotal = seqColAEntityMap.get("lengths").size();
-        Integer seqColBTotal = seqColBEntityMap.get("lengths").size();
-        comparisonResult.putIntoElements("total", "a", seqColATotal);
-        comparisonResult.putIntoElements("total", "b", seqColBTotal);
+        // "array_elements" attribute | "a"
+        for (String attribute: seqColAAttributeSet) {
+            // Looping through each attribute of seqcolA, Eg: "sequences", "lengths", etc...
+            comparisonResult.putIntoArrayElements("a", attribute, seqColAEntityMap.get(attribute).size());
```

Comment:
I was going to say that we should check the that `seqColAEntityMap.get(attribute)` returns a `List` but it is currently enforced anyway.
We may need to support in the future sequence collection that contains non-list attributes. But that's for another issue.

Reply:
Absolutely, I was going to mention that in the next meeting.
Currently, when providing a single value attribute, the service throws an exception and I didn't know how to handle it (how the spec species how should we handle it).